fixieai/cli/agent/commands.py

```diff
@@ -312,7 +312,8 @@ def _configure_venv(
 
     venv_path = os.path.join(agent_dir, ".venv")
     console.print(f"Configuring virtual environment in {venv_path}")
-    venv.create(venv_path, with_pip=True)
+    if not os.path.exists(venv_path):
+        venv.create(venv_path, with_pip=True)
```

Comment:
Was this actually failing for you? I never saw failures from this even when it already existed

Reply:
It fails if we run directly from this repo `poetry run fixie deploy`, but not with `fixie deploy` as an installed pip. I can't tell if this is python/venv version thing, or just running from repo causes this failure. Note that `poetry run fixie deploy` does not fail if `.venv` doesn't exist (the first time). When it fails, the error is:
```
➜  fixie-sdk git:(hessam/fix-tarballing) ✗ poetry run fixie deploy
Configuring virtual environment in ./.venv
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/hessam/projects/fixie/fixie-sdk/.venv/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/Users/hessam/projects/fixie/fixie-sdk/.venv/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/Users/hessam/projects/fixie/fixie-sdk/.venv/lib/python3.9/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/hessam/projects/fixie/fixie-sdk/.venv/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/hessam/projects/fixie/fixie-sdk/.venv/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/Users/hessam/projects/fixie/fixie-sdk/.venv/lib/python3.9/site-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/Users/hessam/projects/fixie/fixie-sdk/fixieai/cli/agent/commands.py", line 504, in deploy
    python_exe, agent_env = _configure_venv(console, agent_dir)
  File "/Users/hessam/projects/fixie/fixie-sdk/fixieai/cli/agent/commands.py", line 315, in _configure_venv
    venv.create(venv_path, with_pip=True)
  File "/Users/hessam/.pyenv/versions/3.9.13/lib/python3.9/venv/__init__.py", line 423, in create
    builder.create(env_dir)
  File "/Users/hessam/.pyenv/versions/3.9.13/lib/python3.9/venv/__init__.py", line 73, in create
    self.setup_python(context)
  File "/Users/hessam/.pyenv/versions/3.9.13/lib/python3.9/venv/__init__.py", line 257, in setup_python
    copier(context.executable, path)
  File "/Users/hessam/.pyenv/versions/3.9.13/lib/python3.9/venv/__init__.py", line 200, in symlink_or_copy
    shutil.copyfile(src, dst)
  File "/Users/hessam/.pyenv/versions/3.9.13/lib/python3.9/shutil.py", line 244, in copyfile
    raise SameFileError("{!r} and {!r} are the same file".format(src, dst))
shutil.SameFileError: '/Users/hessam/projects/fixie/fixie-sdk/.venv/bin/python' and '/Users/hessam/projects/fixie/fixie-sdk/.venv/bin/python' are the same file
```

Reply:
Per our discussion offline, changed .venv to .fixie.venv and checking for existence of the python exec